src/Compatibility/Microsoft.DotNet.PackageValidation/Microsoft.DotNet.PackageValidation.csproj

```diff
@@ -3,10 +3,10 @@
   <PropertyGroup>
     <TargetFrameworks>$(SdkTargetFramework);net472</TargetFrameworks>
     <IsPackable>true</IsPackable>
-    <IsShipping>true</IsShipping>
+    <IsShippingPackage>true</IsShippingPackage>
```

Comment:
why changed it to use `IsShippingPackage` instead?

Reply:
IsShipping is not working as IsShippingPackage is set to false for all the repo. we need to set this true in order to ship it as a shippping asset.

Reply:
Hmm, then I need to add it to the Compatibility project as well.

Reply:
yes